### PR TITLE
add back button to device screen. remove debug back since its confusing

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466097190515-ef58be1a3462-bde276878113/Device_Page-Unlock-ef58be1a3462.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466097190515-ef58be1a3462-bde276878113/Device_Page-Unlock-bde276878113.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466097190515-ef58be1a3462-bde276878113/Device_Page-Revoked-ef58be1a3462.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466097190515-ef58be1a3462-bde276878113/Device_Page-Revoked-bde276878113.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466097190515-ef58be1a3462-bde276878113/Device_Page-Paper-ef58be1a3462.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466097190515-ef58be1a3462-bde276878113/Device_Page-Paper-bde276878113.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466097190515-ef58be1a3462-bde276878113/Device_Page-Normal-ef58be1a3462.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466097190515-ef58be1a3462-bde276878113/Device_Page-Normal-bde276878113.png)  
   Device_Page-Normal comparison

File: shared/router/meta-navigator.render.desktop.js [hunk 1]
```diff
@@ -1,53 +1,20 @@
...
       <div style={{...globalStyles.flexBoxColumn, flex: 1}}>
         {element}
         {!element && Module && <Module {...componentAtTop.props} />}
-        {!hideBack && uri && uri.count() > 1 && <BackButton title='Debug Back' onClick={() => this.onBack()} style={styles.backButton} />}
       </div>
     )
   }
 }
 
-MetaNavigatorRender.propTypes = {
```
Comment (MarcoPolo) [inline shared/router/meta-navigator.render.desktop.js]: If we're killing this, can we replace it with flow types?

Comment (chrisnojima) [inline shared/router/meta-navigator.render.desktop.js]: i'm putting it back.i don't want to go down a whole flow rabbit hole now

**Comment by keybase-ci-visdiff** [comment]
:mag_right: These commits introduced some visual changes:
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466099818202-3caec0d10c21-bd3e8bb5d074/Device_Page-Unlock-3caec0d10c21.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466099818202-3caec0d10c21-bd3e8bb5d074/Device_Page-Unlock-bd3e8bb5d074.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466099818202-3caec0d10c21-bd3e8bb5d074/Device_Page-Revoked-3caec0d10c21.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466099818202-3caec0d10c21-bd3e8bb5d074/Device_Page-Revoked-bd3e8bb5d074.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466099818202-3caec0d10c21-bd3e8bb5d074/Device_Page-Paper-3caec0d10c21.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466099818202-3caec0d10c21-bd3e8bb5d074/Device_Page-Paper-bd3e8bb5d074.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466099818202-3caec0d10c21-bd3e8bb5d074/Device_Page-Normal-3caec0d10c21.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466099818202-3caec0d10c21-bd3e8bb5d074/Device_Page-Normal-bd3e8bb5d074.png)  
   Device_Page-Normal comparison

**Comment by MarcoPolo** [comment]
👍 

Although I imagine us having a wrapper component that puts a back button on any given component. but this is fine for now

**Comment by chrisnojima** [comment]
yeah we'll build that soon, this is just to fix this real quick cause its totally not there now

**Comment by awendland** [comment]
@chrisnojima How do we navigate up from the dumb sheet?

**Comment by chrisnojima** [comment]
we'll have to add a real back button!

**Comment by awendland** [comment]
👍 % if I can escape from the depths of the dumb sheet back to the pastures of the settings screen

**Comment by chrisnojima** [comment]
retest this please

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 723d8a4a4cb0...2bae21d2b936 introduced some visual changes:
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466108247205-723d8a4a4cb0-2bae21d2b936/Device_Page-Unlock-723d8a4a4cb0.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466108247205-723d8a4a4cb0-2bae21d2b936/Device_Page-Unlock-2bae21d2b936.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466108247205-723d8a4a4cb0-2bae21d2b936/Device_Page-Revoked-723d8a4a4cb0.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466108247205-723d8a4a4cb0-2bae21d2b936/Device_Page-Revoked-2bae21d2b936.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466108247205-723d8a4a4cb0-2bae21d2b936/Device_Page-Paper-723d8a4a4cb0.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466108247205-723d8a4a4cb0-2bae21d2b936/Device_Page-Paper-2bae21d2b936.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466108247205-723d8a4a4cb0-2bae21d2b936/Device_Page-Normal-723d8a4a4cb0.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466108247205-723d8a4a4cb0-2bae21d2b936/Device_Page-Normal-2bae21d2b936.png)  
   Device_Page-Normal comparison

**Comment by chrisnojima** [comment]
k back buttons added : @keybase/react-hackers bump

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits a82156a33877...3fabddcd2950 introduced some visual changes:
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466173067362-a82156a33877-3fabddcd2950/Device_Page-Unlock-a82156a33877.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466173067362-a82156a33877-3fabddcd2950/Device_Page-Unlock-3fabddcd2950.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466173067362-a82156a33877-3fabddcd2950/Device_Page-Revoked-a82156a33877.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466173067362-a82156a33877-3fabddcd2950/Device_Page-Revoked-3fabddcd2950.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466173067362-a82156a33877-3fabddcd2950/Device_Page-Paper-a82156a33877.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466173067362-a82156a33877-3fabddcd2950/Device_Page-Paper-3fabddcd2950.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466173067362-a82156a33877-3fabddcd2950/Device_Page-Normal-a82156a33877.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466173067362-a82156a33877-3fabddcd2950/Device_Page-Normal-3fabddcd2950.png)  
   Device_Page-Normal comparison

**Comment by awendland** [comment]
@chrisnojima The dev menu currently isn't scrollable because of a stray, non-flex styled Box in dev-menu.desktop.js. 

These style changes need to be made

*dev-menu.desktop.js*
```jsx
<Box style={{
    ...globalStyles.flexBoxColumn,
    flex: 1
}}>
  <BackButton onClick={() => this.props.navigateUp()} />
  <MenuList items={menuItems} />
</Box>
```
*menu-list.desktop.js*
```jsx
<Box style={{...globalStyles.scrollable, flex: 1}}>
```

**Comment by awendland** [comment]
I'm also getting some propType errors and some setState transition warnings. I'll report momentarily.

**Comment by awendland** [comment]
When trying to navigate from dev menu to Dumb components and when first loading the dev menu
screen shot 2016-06-17 at 3 15 29 pm


When first loading the dev menu

screen shot 2016-06-17 at 3 16 46 pm

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 183de871e9c4...6079f66fb765 introduced some visual changes:
 * Changed: **Device_Page-Unlock** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466433703745-183de871e9c4-6079f66fb765/Device_Page-Unlock-183de871e9c4.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466433703745-183de871e9c4-6079f66fb765/Device_Page-Unlock-6079f66fb765.png)  
   Device_Page-Unlock comparison
 * Changed: **Device_Page-Revoked** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466433703745-183de871e9c4-6079f66fb765/Device_Page-Revoked-183de871e9c4.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466433703745-183de871e9c4-6079f66fb765/Device_Page-Revoked-6079f66fb765.png)  
   Device_Page-Revoked comparison
 * Changed: **Device_Page-Paper** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466433703745-183de871e9c4-6079f66fb765/Device_Page-Paper-183de871e9c4.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466433703745-183de871e9c4-6079f66fb765/Device_Page-Paper-6079f66fb765.png)  
   Device_Page-Paper comparison
 * Changed: **Device_Page-Normal** [(before)](http://keybase-app-visdiff.s3.amazonaws.com/1466433703745-183de871e9c4-6079f66fb765/Device_Page-Normal-183de871e9c4.png) [(after)](http://keybase-app-visdiff.s3.amazonaws.com/1466433703745-183de871e9c4-6079f66fb765/Device_Page-Normal-6079f66fb765.png)  
   Device_Page-Normal comparison

**Comment by chrisnojima** [comment]
@awendland all that stuff is fixed

**Comment by awendland** [comment]
🚢

**Pull request merged**